dockers/base-cuda/Dockerfile

```diff
@@ -34,7 +34,7 @@ SHELL ["/bin/bash", "-c"]
 
 ENV PATH="$PATH:/root/.local/bin"
 
-RUN apt-get update && apt-get install -y --no-install-recommends \
+RUN apt-get clean && apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
This seems strange, you need update before install not clean..

Reply:
Turns out it's CDN issue, some suggest clean and update
but let's wait and see if CDN can solve this

https://github.com/NVIDIA/nvidia-docker/issues/1392
https://github.com/NVIDIA/nvidia-docker/issues/877
https://github.com/NVIDIA/nvidia-docker/issues/1328

Reply:
ok, the problem is in 
```
Reading package lists...
E: Failed to fetch https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64/Packages.gz  File has unexpected size (47871 != 49498). Mirror sync in progress? [IP: 152.195.19.142 443]
   Hashes of expected file:
    - Filesize:49498 [weak]
    - SHA256:332f3ee4e353b8a5e5a2bdd8fdbd47cf140c73822b82b328815f122e09e195a0
    - SHA1:4dc8ef9a3ee3c97b3c26d46e07fdd83997e6880b [weak]
    - MD5Sum:bbff3b9c3462257479d72521ee78ec29 [weak]
   Release file created at: Wed, 23 Sep 2020 22:09:13 +0000
E: Some index files failed to download. They have been ignored, or old ones used instead.
```
not the update step...

Reply:
That comes from this, I think
```
Get:10 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  Packages [49.5 kB]
Err:10 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  Packages
  File has unexpected size (47871 != 49498). Mirror sync in progress? [IP: 152.195.19.142 443]
  Hashes of expected file:
   - Filesize:49498 [weak]
   - SHA256:332f3ee4e353b8a5e5a2bdd8fdbd47cf140c73822b82b328815f122e09e195a0
   - SHA1:4dc8ef9a3ee3c97b3c26d46e07fdd83997e6880b [weak]
   - MD5Sum:bbff3b9c3462257479d72521ee78ec29 [weak]
  Release file created at: Wed, 23 Sep 2020 22:09:13 +0000
```

Reply:
so skip of the `update` is just hiding the NVIDIA package issue not solving it.. :[